clouddriver-google/src/test/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/loadbalancer/UpsertGoogleHttpLoadBalancerAtomicOperationUnitSpec.groovy

```diff
@@ -765,7 +765,7 @@ class UpsertGoogleHttpLoadBalancerAtomicOperationUnitSpec extends Specification
 
       def backendServices = Mock(Compute.BackendServices)
       def backendServicesList = Mock(Compute.BackendServices.List)
-      def bsName = "$LOAD_BALANCER_NAME-${UpsertGoogleHttpLoadBalancerAtomicOperation.BACKEND_SERVICE_NAME_PREFIX}-$PM_SERVICE"
+      def bsName = PM_SERVICE
```

Comment:
Can you drop all of these reassignments and just use the referenced strings directly?